components/App.js

```diff
@@ -42,6 +42,124 @@ export const onclickBindings = {
       menu,
     });
   },
+  'threeD-social-tradeBtn': (e) => {
+    const { menu } = getState();
+    menu.trade.visible = true;
+    menu.trade.toPeer = e.name;
+    menu.trade.fromPeer = loginManager.getAddress();
+    setState({
+      menu,
+    });
+  },
+  'threeD-trade-agreement': (e) => {
+    const { menu } = getState();
+    menu.trade.agreement = !menu.trade.agreement;
+    setState({ menu });
+  },
+  'threeD-trade-header-backBtn': (e) => {
+    const { menu } = getState();
+    menu.trade = {
+      visible: false,
+      toPeer: null,
+      fromPeer: null,
+      selectedItem: null,
+      agreement: false,
+      inventoryPage: 0,
+      peersPage: 0,
+    }
+    setState({
+      menu,
+    });
+  },
+  'threeD-trade-cancel': (e) => {
+    const { menu } = getState();
+    menu.trade = {
+      visible: false,
+      toPeer: null,
+      fromPeer: null,
+      selectedItem: null,
+      agreement: false,
+      inventoryPage: 0,
+      peersPage: 0,
+    }
+    setState({
+      menu,
+    });
+  },
+  'threeD-trade-peers-card': (e) => {
+    console.log('hello', e)
+    const { menu } = getState();
+    menu.trade.toPeer = e.name;
+    setState({ menu });
+  },
+  'threeD-trade-inventory-back': (e) => {
+    const { menu } = getState();
+    menu.trade.inventoryPage > 0 ? --menu.trade.inventoryPage : null;
+    setState({ menu });
+  },
+  'threeD-trade-inventory-forward': (e) => {
+    const { menu } = getState();
+    ++menu.trade.inventoryPage;
```

Comment:
Our convention is postfix `++`.

Reply:
Also, is this breaking past the total number of pages of content? It could be capped.

Reply:
fixed